cluster-api/src/main/java/io/scalecube/cluster/ClusterConfig.java

```diff
@@ -152,28 +152,67 @@ public ClusterConfig metadataDecoder(MetadataDecoder metadataDecoder) {
     return c;
   }
 
+  /**
+   * Deprecated since {@code 2.4.10} in favor of {@link #mappedHost()}.
+   *
+   * @return member host
+   */
+  @Deprecated
   public String memberHost() {
-    return memberHost;
+    return mappedHost;
   }
 
   /**
-   * Sets a memberHost.
+   * Deprecated since {@code 2.4.10} in favor of {@link #mappedHost(String)}.
    *
    * @param memberHost member host
    * @return new {@code ClusterConfig} instance
    */
+  @Deprecated
   public ClusterConfig memberHost(String memberHost) {
     ClusterConfig c = clone();
-    c.memberHost = memberHost;
+    c.mappedHost = memberHost;
+    return c;
+  }
+
+  /**
+   * Returns mappedHost. {@code mappedHost} is a config property for container environments, it's
+   * being set for advertising to scalecube cluster some connectable hostname which maps to
+   * scalecube transport's hostname on which scalecube transport is listening.
+   *
+   * @return mapped host
+   */
+  public String mappedHost() {
+    return mappedHost;
+  }
+
+  /**
+   * Setter for mappedHost. {@code mappedHost} is a config property for container environments, it's
+   * being set for advertising to scalecube cluster some connectable hostname which maps to
+   * scalecube transport's hostname on which scalecube transport is listening.
+   *
+   * @param mappedHost mapped host
+   * @return new {@code ClusterConfig} instance
+   */
+  public ClusterConfig mappedHost(String mappedHost) {
```

Comment:
maybe the name is better to be exportedHostName / externalHost / containerHost

mapped name is a term taken from what world?

for example in dockers its export
in routers its mapping?

Reply:
Are you good with `containerHost` and `containerPort`?

Reply:
For me, `mapped*` was better